CMakeLists.txt

```diff
@@ -369,19 +371,21 @@ set(CommonWindows
 	Common/stdafx.h)
 source_group(Windows FILES ${CommonWindows})
 
-set(CommonVulkan ${CommonExtra}
-	Common/Vulkan/SPIRVDisasm.cpp
-	Common/Vulkan/SPIRVDisasm.h
-	Common/Vulkan/VulkanContext.cpp
-	Common/Vulkan/VulkanContext.h
-	Common/Vulkan/VulkanDebug.cpp
-	Common/Vulkan/VulkanDebug.h
-	Common/Vulkan/VulkanImage.cpp
-	Common/Vulkan/VulkanImage.h
-	Common/Vulkan/VulkanLoader.cpp
-	Common/Vulkan/VulkanLoader.h
-	Common/Vulkan/VulkanMemory.cpp
-	Common/Vulkan/VulkanMemory.h)
+if(NOT WIIU)
```

Comment:
It'd be nice if we could actually put WIIU ifdefs via ppsspp_config.h in all these files, and have CMake include them always.

-[Unknown]